src/styles/text/label_line.js

```diff
@@ -89,28 +90,18 @@ export default class LabelLine extends Label {
         return [ p1, p2 ];
     }
 
-    computeBBox (size) {
+    computeAABB (size) {
         let upp = Geo.units_per_pixel;
 
         let merc_width = this.size.text_size[0] * upp;
```

Comment:
This is a pre-existing issues, but these values are not actually mercator, so these are misleading variables names. It is an arbitrary scale/coordinate system local to the tile (the `Geo.units_per_pixel` provides that conversion). So we can just use some more generic name here. We should also rename (or just eliminate) the `Utils.pixelToMercator` method, since it is similar misnomer: https://github.com/tangrams/tangram/blob/obb/src/utils/utils.js#L437